opentelemetry-api/TrackException/pom.xml

```diff
@@ -10,6 +10,10 @@
     <modelVersion>4.0.0</modelVersion>
 
     <artifactId>TrackException</artifactId>
+    <packaging>pom</packaging>
+    <modules>
+        <module>Advanced</module>
+    </modules>
```

Comment:
I find modules which are both real modules and parent modules a bit awkward, i'd suggest either including it in `TrackException` module, or making it a sibling of the other `Track*` module

Reply:
i will make it a sibling

Reply:
i will keep it in TrackException.. seem to be simpler